nightingale_ros_bridge/launch/interface.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
```

Comment:
Has this launch file been tested yet? Last time I used it it didn't seem to work and I had to launch the nodes individually

Reply:
I think so, I'm running "roslaunch nightingale_ros_bridge interface.launch" and getting things communicating as you saw today